ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/show/tables/ShowTablesFormatter.java

```diff
@@ -51,7 +52,7 @@ public static ShowTablesFormatter getFormatter(HiveConf conf) {
 
   public abstract void showTables(DataOutputStream out, List<String> tables) throws HiveException;
 
-  abstract void showTablesExtended(DataOutputStream out, List<Table> tables) throws HiveException;
+  abstract void showTablesExtended(DataOutputStream out, List<Pair<String, String>> tables) throws HiveException;
```

Comment:
why is that change? if you'd like to avoid using fat class, maybe try org.apache.hadoop.hive.metastore.api.Table?

Reply:
`SHOW TABLES EXTENDED` only outputs table name and table type, so we do not need the `Table` objects here.